web/gds-user-ui/src/locales/fr/messages.po

```diff
@@ -1743,60 +1690,56 @@ msgstr ""
 
 #: src/components/Section/SearchDirectory.tsx:159
 msgid "TESTNET DIRECTORY RECORD"
-msgstr ""
+msgstr "FICHE D'ANNUAIRE DE TESTNET"
 
 #: src/components/ReviewSubmit/index.tsx:118
 msgid "TESTNET SUBMISSION"
-msgstr ""
+msgstr "SOUMISSION DE TESTNET"
 
 #: src/components/CertificateManagement/index.tsx:52
 msgid "TRISA Certificates"
 msgstr ""
 
-#: src/components/OrganizationProfile/TrisaDetail.tsx:20
-#~ msgid "TRISA Details"
-#~ msgstr ""
-
 #: src/components/TrisaImplementationForm/index.tsx:68
 msgid "TRISA Endpoint"
-msgstr ""
+msgstr "Point de terminaison TRISA"
 
 #: src/components/TrisaImplementation/index.tsx:77
 msgid "TRISA Endpoint: MainNet"
-msgstr ""
+msgstr "Point de terminaison TRISA: MainNet"
 
 #: src/components/TrisaImplementation/index.tsx:72
 msgid "TRISA Endpoint: TestNet"
-msgstr ""
+msgstr "Point de terminaison TRISA: TestNet"
 
 #: src/components/OpenSourceResources/index.tsx:15
-msgid "TRISA Githubs repo"
```

Comment:
@elysee15 @masskoder another typo - can you please fix in the app code?

It should be `TRISA Github repo`

Reply:
Fixed in this PR.